portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase

```diff
@@ -0,0 +1,95 @@
+@component-name = "portal-headless-frontend-infrastructure"
```

Comment:
I think `@component-name = "portal-frontend-infrastructure"` makes more sense. I don't see this testing headless api.